pkg/service/discord/discord.go

```diff
@@ -427,3 +429,25 @@ func (d *Discord) NotifyMemberJoin(discordID, avatar, jlChannelID string, userCo
 	}
 	return nil
 }
+
+func (d *Discord) SendFeedback(req *request.UserFeedbackRequest) error {
+	title := "Feedback recieved"
+	if req.Command != "" {
+		title += fmt.Sprintf(" for command: %s", req.Command)
+	} else {
+		title += "."
+	}
+	msg := &discordgo.MessageEmbed{
+		Title:       title,
+		Footer:      &discordgo.MessageEmbedFooter{Text: req.Username, IconURL: req.Avatar},
+		Description: fmt.Sprintf("<@%s> has something to say\n`%s`", req.DiscordID, req.Feedback),
```

Comment:
no i don't think it's necessary because i don't want to accidentally remove users special characters, but also don't want them to use any markdown so i think keeping everything in `this` is appropriate